src/ui/LiveSplit.tsx

```diff
@@ -261,31 +268,35 @@ export class LiveSplit extends React.Component<{}, State> {
         const viewContainerClassName = `view-container ${routeClassMap[this.state.menu.kind]}`;
 
         return (
-            <Sidebar
-                sidebar={sidebarContent}
-                docked={this.state.isDesktop}
-                open={this.state.sidebarOpen}
-                transitions={this.state.sidebarTransitionsEnabled}
-                onSetOpen={((e: boolean) => this.onSetSidebarOpen(e)) as any}
-                sidebarClassName="sidebar"
-                contentClassName={contentClassName}
-                overlayClassName="sidebar-overlay"
-            >
-                {
-                    !this.state.isDesktop &&
-                    !this.state.sidebarOpen &&
-                    <button
-                        className="sidebar-button fa fa-bars"
-                        onClick={((e: boolean) => this.onSetSidebarOpen(e)) as any}
-                    />
-                }
-                <div
-                    className={viewContainerClassName}
-                    ref={this.containerRef}
+            <>
+                <input id="file-input" type="file" ref={this.fileInputElement} />
+                <a id="file-output" ref={this.fileOutputElement} />
```

Comment:
I get why this is done this way, but I really don't like it :(

Reply:
What's the reason that you don't like it?

Reply:
Because the fact that any html elements even exist at all should be an implementation detail of those file utility functions.

Reply:
Looks like it's easy enough to hack around it in the test code without having to restructure our actual application code: https://github.com/LiveSplit/LiveSplitOne/blob/3d45d1c6a66438102ec2cf8f518b83f3c607e670/test/rendering-test.js#L48-L56